### ISSUE TITLE
Update demo benchmarks to FastDoom 0.9.3

### ISSUE THREAD

**Issue by RamonUnch** [viti95/FastDoom#136]
There has been some progress on the performances of FastDoom however the README.md contains outdated benchmarks data from 0.2
I think the table should be updated with newer version. Also maybe you could add a line for a 386 machine? I think you have one in working order right?

https://github.com/viti95/FastDoom/blob/275c1e513882aee174b719e0d07dbefb9f0e3fab/README.md?plain=1#L34-L39

**Comment by viti95** [comment]
Yeah I should update the whole readme, there has been lots of changes since then 😅

I'll add 386SX, 386DX and 486 cpu's to the table, maybe also something more powerful.

**Comment by SpitFire-666** [comment]
I've got a 386SX @ 16MHz and a 386DX @ 20MHz - I'd be happy to benchmark, would just need the commands/demo file to make sure the tests are accurate

**Comment by viti95** [comment]
Nice, I'll upload a batch file to benchmark FastDoom easily, probably next Tuesday.

**Comment by viti95** [comment]
I've created a new batch to benchmark FastDoom along some changes in FastDoom code to be able to generate CSV reports. This is way is much easier to generate reports and charts, but will have to wait to 0.9.5 release.

**Comment by SpitFire-666** [comment]
Cool, i grabbed the .bat but unfortunately I couldn't get it to save the results to the CSV file 🤔

It ran the demos and the showed the results on the screen but the CSV file only contained the echoed info as per this :

![image](https://user-images.githubusercontent.com/38451588/227670834-f80661bf-8f82-44b8-923e-2861fb3dc97a.png)

**Comment by viti95** [comment]
This is normal, the updated CSV code won't be available until FastDoom 0.9.5, but I can upload a testing executable if you wish.
